modules/javafx.graphics/src/main/java/javafx/css/Stylesheet.java

```diff
@@ -63,8 +64,9 @@ public class Stylesheet {
      * binary stream changes. This number does not correlate with JavaFX versions.
      * Version 5: persist @font-face
      * Version 6: converter classes moved to public package
+     * Version 7: media queries
```

Comment:
this, in my opinion, deserves a constant (it does not have to be public).

Reply:
I'd rather do this in a refactor PR with the rest of the magic numbers.